<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-3ULC79B" ignored="0" name="BankingApp Test(failed)" tests="4" failures="3" timestamp="2022-12-15T18:39:17 EST" time="10.312" errors="0">
  <testcase name="loginTest" time="0.758" classname="com.testcases.TC_LoginTest"/>
  <testcase name="loginDDT" time="0.715" classname="com.testcases.TC_Login_DDTest">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.testcases.TC_Login_DDTest.loginDDT(TC_Login_DDTest.java:30)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="0.711" classname="com.testcases.TC_Login_DDTest">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.testcases.TC_Login_DDTest.loginDDT(TC_Login_DDTest.java:30)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="0.777" classname="com.testcases.TC_Login_DDTest">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.testcases.TC_Login_DDTest.loginDDT(TC_Login_DDTest.java:30)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
